mlc_config.json

```diff
@@ -1,7 +0,0 @@
-{
-  "ignorePatterns": [
-    {
-      "pattern": "^changes_0.1.0.md"
-    }
-  ]
-}
```

Comment:
Let's add these also to broken-links checker config and remove the comment from changes file?

Reply:
Okay, this also does not make sense as discussed. It is only for PK itself.